apps/crn-server/test/helpers/users.ts

```diff
@@ -1,43 +1,7 @@
-import { RestUser, SquidexRest, User } from '@asap-hub/squidex';
 import Chance from 'chance';
-import { appName, baseUrl } from '../../src/config';
-import { getAuthToken } from '../../src/utils/auth';
 
-const users = new SquidexRest<RestUser>(getAuthToken, 'users', {
-  appName,
-  baseUrl,
-});
 const chance = new Chance();
 
-export const createUser = (overwrites?: Partial<User>): Promise<RestUser> => {
-  const userData: User = {
-    onboarded: true,
-    dismissedGettingStarted: false,
-    firstName: chance.first(),
-    lastName: chance.last(),
-    jobTitle: chance.suffix({ full: true }),
-    orcid: createRandomOrcid(),
-    institution: chance.company(),
-    email: chance.email(),
-    biography: chance.paragraph({ sentence: 3 }),
-    role: 'Grantee',
-    avatar: [],
-    expertiseAndResourceTags: [],
-    questions: [],
-    teams: [],
-    connections: [],
-    labs: [],
-    ...overwrites,
-  };
-
-  const user = Object.entries(userData).reduce((acc, [key, value]) => {
-    acc[key] = { iv: value };
-    return acc;
-  }, {} as { [key: string]: { iv: unknown } });
-
-  return users.create(user as RestUser['data']);
-};
-
 export const createRandomOrcid = () =>
```

Comment:
is this being used anywhere?

Reply:
it is not but i think we might leave it, this could POTENTIALLY be used soon so